### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by c-nichols**
Python 3 Support -- some (very) light testing indicates that basic functionality is there on 2.7 and 3.3.  Basically I made a few requests to the product endpoint... could definitely use some more thorough exercise :)  
Added six as a requirement.
What do you think?

### CODE REVIEW

**Comment by bloodywing** [comment]
Using the Python3 Version a bit more now and fixed some stuff: d5d68554312d17beb0da0702373a2d9efa2c59aa

Haven't heard something from paulo for a while now. And I'm fine with six, saw it in other big libraries too.

**Comment by czpython** [comment]
yeah been swamped with work.. :+1: from me to use six, just gotta get some time to review it properly..

File: mws/mws.py [hunk 1]
```diff
@@ -195,7 +203,7 @@ def make_request(self, extra_data, method="GET", **kwargs):
             # When retrieving data from the response object,
             # be aware that response.content returns the content in bytes while response.text calls
             # response.content and converts it to unicode.
-            data = response.content
+            data = response.text
```
Comment (czpython) [inline mws/mws.py]: I remember calling response.text on python 2.7 threw some encoding errors, think it's best to check here if using python 3 or 2 and based on that accessing content or text

Comment (GriceTurrble) [inline mws/mws.py]: The `response.content` is the binary response content. If that's what `DataWrapper` needs, probably best to leave it alone.

http://docs.python-requests.org/en/latest/user/quickstart/#response-content

**Comment by ibotty** [comment]
czpython, is your review complete? It might make sense to rebase and merge?

**Comment by ynalcakan** [comment]
This support works for me too, but I am getting an error something like this in mws.py:

'''
--> 187         request_description = '&'.join(['%s=%s' % (k, quote(params[k], safe='-_.~')) for k in sorted(params)])

/usr/lib/python3.4/urllib/parse.py in quote_from_bytes(bs, safe)
719         raise TypeError("quote_from_bytes() expected bytes")

'''
TypeError: quote_from_bytes() expected bytes 

and I couldn't find the solution for this error.

**Pull request closed without merging**